backend/src/main/java/com/znane_groby/backend/controller/CityController.java

```diff
@@ -0,0 +1,51 @@
+package com.znane_groby.backend.controller;
```

Comment:
Myślę że możesz założyć że w javie piszemy tylko backend, więc nie ma sensu tego pisać w pakiecie. Jeśli juz to raczej 'app'.

Reply:
oki